cypress/e2e/App.cy.tsx

```diff
@@ -1,17 +1,22 @@
+import { WelcomePageTestFunctions } from 'cypress/utils/WelcomePage/WelcomePageTestFunctions';
 import { MainPanelTestFunctions } from 'cypress/utils/MainPanel/MainPanelTestFunctions';
 
 const MainPanel = new MainPanelTestFunctions();
 
+const WelcomePage = new WelcomePageTestFunctions();
+
 describe('Root.cy.tsc', () => {
   beforeEach(() => {
     cy.visit('/');
   });
 
-  it('hello world', () => {
-    cy.getBySelector('hello-world-p').contains('Hello World');
+  it('render welcomePage', () => {
+    WelcomePage.testWelcomePageContent();
   });
 
-  it('render MainPanel', () => {
+  it.only('welcomePage click', () => {
```

Comment:
change test name to something like `render MainPanel after WelcomePage click` because you don't only test the actual click but also what's happening after the click